web/src/user_profile.js

```diff
@@ -319,6 +321,27 @@ export function get_custom_profile_field_data(user, field, field_types) {
     return profile_field;
 }
 
+function render_profile_fields(user) {
+    const $custom_profile_field = $("#content");
+    const field_types = page_params.custom_profile_field_types;
+
+    const profile_fields = page_params.custom_profile_fields
```

Comment:
You would need to update this to since the `page_params` object has now been split.

Reply:
Done.